### ISSUE TITLE
[DatePicker] Behaves as mobile on Ubuntu Desktop

### ISSUE THREAD

**Issue by jardelnovaes** [mui/mui-x#4447]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

DatePicker behaves as mobile on Ubuntu Desktop with Chrome (tested with 96 and 99 )
It works without problems on Windows 10 or Ubuntu Desktop with FireFox.

### Expected behavior 🤔

Expected to work as "MobileDatePicker" on mobile devices and as "DesktopDatePicker" on desktops.

### Steps to reproduce 🕹

Steps:

1. Open Chrome from a Ubuntu Desktop (I tried with 20.04)
2. Open URL https://mui.com/components/date-picker/
3. Click on "Basic usage" Field.
4. It will behave like _MobileDatePicker_ instead of _DesktopDatePicker_

### Your environment 🌎

<details>
  <summary>`npx @mui/envinfo`</summary>
<pre>
  System:
    OS: Linux 5.13 Ubuntu 20.04.2 LTS (Focal Fossa)
  Binaries:
    Node: 16.14.1 - ~/.nvm/versions/node/v16.14.1/bin/node
    Yarn: 1.22.5 - /bin/yarn
    npm: 8.5.0 - ~/.nvm/versions/node/v16.14.1/bin/npm
  Browsers:
    Chrome: 99.0.4844.82
    Firefox: 98.0.1
  npmPackages:
    @emotion/react: ^11.8.2 => 11.8.2 
    @emotion/styled: ^11.8.1 => 11.8.1 
    @mui/base:  5.0.0-alpha.73 
    @mui/icons-material: ^5.5.1 => 5.5.1 
    @mui/lab: ^5.0.0-alpha.74 => 5.0.0-alpha.74 
    @mui/material: ^5.5.2 => 5.5.2 
    @mui/private-theming:  5.4.4 
    @mui/styled-engine:  5.5.2 
    @mui/styled-engine-sc: ^5.5.2 => 5.5.2 
    @mui/styles: ^5.5.1 => 5.5.1 
    @mui/system:  5.5.2 
    @mui/types:  7.1.3 
    @mui/utils:  5.4.4 
    @mui/x-data-grid: ^5.6.1 => 5.6.1 
    @types/react: ^17.0.3 => 17.0.14 
    react: 17.0.2 => 17.0.2 
    react-dom: 17.0.2 => 17.0.2 
    typescript: ^4.2.4 => 4.3.5 
</pre>
</details>

**Tested with:**
@mui/lab  v5.0.0-alpha.74
Ubuntu 20.04 
Chrome 96.0.4664.110 and 99.0.4844.82-1

**Comment by flaviendelangle** [comment]
@jardelnovaes We use the following media query to determine if we are on a mobile or a desktop `'@media (pointer: fine)'`

Which type of device are you using ?
I am also on Ubuntu on a desktop computer and it works.

**Comment by jardelnovaes** [comment]
@flaviendelangle, same Ubuntu 20.04 Desktop, left Firefox, right Chrome.

![image](https://user-images.githubusercontent.com/20839085/163023296-62f149cb-0356-4d29-8b86-d43988014503.png)

**Comment by flaviendelangle** [comment]
![image](https://user-images.githubusercontent.com/3309670/163023699-3a86706b-e2aa-4e0c-b63e-01acd6680176.png)

Ubuntu 21.10 Chrome

If you could try on Ubuntu 21.10 it would help me a lot

**Comment by jardelnovaes** [comment]
@flaviendelangle I don't have a Desktop with Ubuntu 21.10, I'll need some time to provide this environment, but it's not the real environment where the app is used.

I was able to test it on another environment (other company) with Ubuntu 20.04 and Plasma KDE and the result was the same.
The "original" test was with Gnome.

I'll let you know when I was able to test on Ubuntu 21.10.

Unfortunately I'm not able to force the about 500 desktops to run with Ubuntu 21.10. So even after this test it still would need to work on Ubuntu 20.04.

**Comment by jardelnovaes** [comment]
@flaviendelangle I've downloaded the official iso of Ubuntu 21.10 and tried to install it on latest version of Virtual Box, unfortunately the installation freezes during hardware configuration steps (more than 50% of installation progress bar)

I've thought it would be a fast option for the test, I was wrong. 

I'll try other way to provide this test.

I've tested with **Ubuntu 20.04 and Chrome 100.0.4896.88**, the problem still happen.
